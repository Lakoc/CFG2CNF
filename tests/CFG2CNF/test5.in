S,A,B
a,b
S
S->aA