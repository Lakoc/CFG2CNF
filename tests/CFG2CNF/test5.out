S,A,B,a'
a,b
S
S->a'A
a'->a